internal/wasmldr/src/main.rs

```diff
@@ -72,7 +72,7 @@ fn main() {
     // (safely, securely) send logs. Might need our own logger for that..
     env_logger::Builder::from_default_env().init();
 
-    info!("version {} starting up", env!("CARGO_PKG_VERSION"));
+    
```

Comment:
Don't add an extra blank line here. Just remove the old line.

Reply:
Sure, let me make the changes

Reply:
@Deepansharora27 Make sure you run `cargo fmt` before committing.

Reply:
Closing this PR

Reply:
@Deepansharora27 In the future, it is better not to create a new PR since this loses the conversation flow. Just force push to your PR's branch.

Reply:
Actually I was thinking to make a force push earlier but subconsciously I was thinking that it would not be good to go with a force push. 
But I will keep in mind for the future 
Apologies for this